### ISSUE TITLE
New prototype in which() breaks existing code passing in a list

### ISSUE THREAD

**Issue by jmdh** [plicease/File-Which#6]
This used to work:

```
sub foo {
    my $self = shift;
    return File::Which::which( @_ );
}
```

However since commit 7155144d9a7756104d240dccc90ee6b7e5eb9435 it fails, since `@_` is being coerced to a scalar. This pitfall is described in http://perldoc.perl.org/perlsub.html#Prototypes.

To avoid breaking existing code, it's probably better to remove the prototype from this sub.

Thanks for your work on File::Which!

**Comment by plicease** [comment]
sorry that was a bad idea.  Should be fixed in 1.18.

**Comment by jmdh** [comment]
Thanks for fixing this so quickly!
